### PR TITLE
SVE Change of packaging again

### PR DESCRIPTION
**Pull request by politas**
I hope that setting a conflict on a version range works. Don't know whether this PR will test properly.

### CODE REVIEW

File: NetKAN/StockVisualEnhancements.netkan [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ],
+    "depends": [
+        { "name": "SVE-Textures" },
+        { "name": "Scatterer" },
+        { "name": "EnvironmentalVisualEnhancements" },
+        { "name": "ModuleManager"}
+    ],
+    "recommends": [
+        { "name": "DistantObject" },
+        { "name": "PlanetShine" },
```
Comment (Olympic1) [inline NetKAN/StockVisualEnhancements.netkan]: remove comma

**Comment by Olympic1** [comment]
Build successful, so I think the range is fine, will do some testing though. Other then that looks good.

**Comment by Olympic1** [comment]
After some extensive testing it doesnt like the version range. If I deleted them, it works.

The download also includes a `SVE-Sunflare` config, so that would need to be added as a `provides` too.

**Comment by politas** [comment]
It works without the ranges on the conflicts? But it depends on one of those! I don't think we can depend on a version range for a metapackage, either.

**Comment by Olympic1** [comment]
I think it needs the scatterer configs/sunflares, but I'm not sure. It modifies the sunflare one.
@Galileo88 can you elaborate.

**Comment by Galileo88** [comment]
Yes, it needs scatterer and its sun flare.

**Comment by politas** [comment]
@Galileo88 , so the SVE_Scatterer folder is providing a sunflare configuration for Scatterer?

**Pull request merged**